content/millennium/dstu2/scheduling/slot.md

```diff
@@ -94,6 +95,12 @@ Notes:
 
 * The search operation will only return free slots which are available to be booked. However, booked slots may still be retrieved when using the `_id` parameter.
 
+* The `_include` parameter may be provided once. Example: `_include=Slot:schedule`
```

Comment:
mention that Slot:schedule is the only supported value

Reply:
https://github.com/cerner/fhir.cerner.com/pull/256/commits/eb14ba57695d47945c5d478cd6afdf31bf97b788#diff-5af5ea7f1f259fa879893204fdd065b5R98